libraryversions.toml

```diff
@@ -20,7 +20,7 @@ CAMERA_TESTING = "1.0.0-alpha01"
 CARDVIEW = "1.1.0-alpha01"
 CAR_APP = "1.7.0-alpha01"
 COLLECTION = "1.4.0"
-COMPOSE = "1.7.0-beta05"
+COMPOSE = "1.7.0-beta06"
 COMPOSE_COMPILER = "1.5.4"
```

Comment:
I see some changes from the compiler were included. Is this version in sync?

Reply:
We don't use `compiler`, but we usually included it in Compose updates.

But it isn't aligned with Compose, so better to start excluding it.

P.S. we probably need scripts for copy-paste folders as well, to pick the right folders 🫠.